sdk/monitor/azure-monitor-opentelemetry-autoconfigure/CHANGELOG.md

```diff
@@ -3,13 +3,20 @@
 ## 1.1.0-beta.1 (Unreleased)
```

Comment:
The Changelog looks out of order to me.

I think it's okay if we skip a beta and jump to stable, but I think the "1.1.0-beta.1" needs to be removed.
I would expect the current release to be at the top of this file.

Reply:
I see that the original commit did exactly what I was expecting, but still had an error.
Is there a version number in any of the project files? The CI may be expecting the project and Changelog to be in sync.

Reply:
I found the previous automatic PR: https://github.com/Azure/azure-sdk-for-java/pull/44545/files#diff-cdd487e0c2f348eb166adbec88e99af25536746cfcdae6cef22c8b7016b81725

There's a pom.xml file that needs to match the version we want to release.